containers/Investment/Investor/index.tsx

```diff
@@ -46,8 +46,10 @@ class InvestorView extends React.Component<Props, State> {
   }
 
   componentDidMount() {
-    const { loadPool, tinlake } = this.props
```

Comment:
Is there a reason why you don't define local variable? Just curious, it makes the code a bit less readable in my view.

Reply:
I think it makes it more readable actually, as you can then easily see where the variables are coming from, which I think is important information for the code. But that's definitely a subjective opinion ;-)